client/vue.config.js

```diff
@@ -11,20 +12,20 @@ module.exports = {
   lintOnSave: false,
   publicPath: process.env.VUE_APP_STATIC_PATH,
   configureWebpack: {
-    module: {
-      rules: [
-        {
-          test: /\.worker\.js$/,
-          include: /node_modules(\/|\\)vtk\.js(\/|\\)/,
-          use: [
-            {
-              loader: 'worker-loader',
-              options: { inline: true, fallback: false },
-            },
-          ],
-        },
-      ],
-    },
+    // module: {
```

Comment:
Should this be removed instead of just commented-out?

Reply:
Removed in [9d48ec7](https://github.com/OpenImaging/miqa/pull/346/commits/9d48ec735b2d7e9961e0a2d063c7fd454e826d78)